modelon/impact/client/experiment_definition/extension.py

```diff
@@ -231,7 +235,7 @@ def to_dict(self):
             simulate_ext.to_dict()
 
         """
-        ext_dict = {}
+        ext_dict: Dict[str, Any] = {}
```

Comment:
This seems to result in a pyright error bellow on line 259, not sure why it gives an error or if this is correct but would be nice if we could fix it.